### PR TITLE
Displayed the updated date information in product details

### PR DESCRIPTION
**Pull request by NitishAroraKPMG**
## Description

Show the updated fare triangle modified date

## Testing instructions

Edit the fare triangle and check the modified date.

### CODE REVIEW

File: repos/fdbt-site/src/pages/api/csvUpload.ts [hunk 1]
```diff
@@ -342,7 +344,9 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
                 // put the now updated matching json into s3
```
Comment (RobCattonKPMG) [inline repos/fdbt-site/src/pages/api/csvUpload.ts:344]: I think this comment might need amending as its not updating S3, its amending the database products table.

File: repos/fdbt-site/src/pages/products/productDetails.tsx [hunk 2]
```diff
@@ -301,7 +303,11 @@ const createProductDetails = async (
     if ('lineId' in ticket) {
         productDetailsElements.push({
             name: 'Fare triangle',
-            content: ['You created a fare triangle'],
+            content: [
+                fareTriangleModified
+                    ? `Updated: ${convertDateFormat(fareTriangleModified?.toString())}`
```
Comment (RobCattonKPMG) [inline repos/fdbt-site/src/pages/products/productDetails.tsx:308]: shouldnt need the ? on the end of fareTriangleModified, as we know its not undefined here.

File: repos/fdbt-site/tests/pages/products/productDetails.test.tsx [hunk 3]
```diff
@@ -196,6 +196,7 @@ describe('myfares pages', () => {
                     lineId: 'q2gv2ve',
                     passengerTypeId: 9,
                     csrfToken: '',
+                    fareTriangleModified: undefined,
```
Comment (RobCattonKPMG) [inline repos/fdbt-site/tests/pages/products/productDetails.test.tsx:199]: Should we add a render test, and a props test, for when fareTriangleModified isnt undefined?

File: repos/fdbt-site/tests/pages/api/csvUpload.test.ts [hunk 4]
```diff
@@ -21,14 +21,22 @@ import {
...
+jest.mock('../../../src/data/auroradb');
 jest.spyOn(s3, 'putDataInS3');
 jest.spyOn(userData, 'putUserDataInProductsBucketWithFilePath');
 
 describe('csvUpload', () => {
     const loggerSpy = jest.spyOn(logger, 'warn');
     const getFormDataSpy = jest.spyOn(fileUpload, 'getFormData');
     const updateSessionAttributeSpy = jest.spyOn(sessions, 'updateSessionAttribute');
+    const updateProductFareTriangleModifiedByNocCodeAndIdSpy = jest.spyOn(
+        auroradb,
```
Comment (RobCattonKPMG) [inline repos/fdbt-site/tests/pages/api/csvUpload.test.ts:35]: Do we have any new tests for the new API functionality?

Comment (NitishAroraKPMG) [reply repos/fdbt-site/tests/pages/api/csvUpload.test.ts:35]: no as we are just updating the current date in products table for fareTriangleModified

File: repos/fdbt-site/tests/pages/products/productDetails.test.tsx (hunk 3 above)
Comment (NitishAroraKPMG) [reply repos/fdbt-site/tests/pages/products/productDetails.test.tsx:236]: done

**Review by RobCattonKPMG — APPROVED** [review]

**Pull request merged**